docs/sources/reference/components/prometheus/prometheus.write.queue.md

```diff
@@ -117,6 +116,31 @@ Name                   | Type     | Description
 `insecure_skip_verify` | `bool`   | Disables validation of the server certificate.           |         | no
 `key_pem`              | `secret` | Key PEM-encoded text for client authentication.          |         | no
 
+
+### parallelism block
+
+Name            | Type       | Description                                                                                                                        | Default | Required
+----------------|------------|------------------------------------------------------------------------------------------------------------------------------------|---------|---------
+`drift_scale_up_seconds`      | `int`      | The maximum amount of time between the timestamps of incoming signals and outgoing signals before increasing desired connections.  | `60`    | no
+`drift_scale_down_seconds`    | `int`      | The minimum amount of time between the timestamps of incoming signals and outgoing signals before descreasing desired connections. | `30`    | no
+`max_connections`    | `uint`     | The maximum number of desired connections.                                                                                         | `50`    | no
+`min_connections`    | `uint`     | The minimum number of desired connections.                                                                                         | `2`     | no
+`network_flush_interval`    | `duration` | The length of time that network successes and failures are kept for determining desired connections.                               | `1m`    | no
+`desired_connections_lookback`    | `duration` | The length of time that previous desired connections are kept for determining desired connections.                                 | `5m`    | no
+`desired_check_interval`    | `duration` | The length of time that between checking for desired connections.                                                                  | `5s`    | no
+`allowed_network_error_percent`    | `float`    | The allowed error rate before scaling down.                                                                                        | `0.50`   | no
```

Comment:
This is 0.5% = 0.005 or is it 50% = 0.5? Just to be sure ;)

Reply:
0.5 would mean allow 50 percent of failures to happen before scaling down.